### PR TITLE
[fix](test) Fix some testcases

### PR DESCRIPTION
**Pull request by zclllyybb**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zclllyybb** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]
LGTM

File: regression-test/suites/ddl_p0/test_create_table_auto_partition.groovy [hunk 1]
```diff
@@ -22,9 +22,9 @@
 suite("test_create_table_auto_partition") {
     def testTable = "test_create_table_auto_partition_table"
 
-    sql "DROP TABLE IF EXISTS ${testTable}"
+    sql "DROP TABLE IF EXISTS test_create_table_auto_partition_table"
```
Comment (morrySnow) [inline regression-test/suites/ddl_p0/test_create_table_auto_partition.groovy:25]: Unexpected changes?

Comment (zclllyybb) [reply regression-test/suites/ddl_p0/test_create_table_auto_partition.groovy:25]: all table name in testcases should avoid use variable. So I think this is proper

Comment (morrySnow) [reply regression-test/suites/ddl_p0/test_create_table_auto_partition.groovy:25]: so should remove `def testTable = "test_create_table_auto_partition_table"`?

Comment (zclllyybb) [reply regression-test/suites/ddl_p0/test_create_table_auto_partition.groovy:23]: yes. fixed. thx~

**Comment by zclllyybb** [comment]
run buildall

**Review by Mryange — APPROVED** [review]

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**